plugin/util.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 the Drone Authors. All rights reserved.
+// Use of this source code is governed by the Blue Oak Model License
+// that can be found in the LICENSE file.
+
+package plugin
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+func writeCard(path, schema string, card interface{}) {
+	data, _ := json.Marshal(map[string]interface{}{
+		"schema": schema,
+		"data":   card,
+	})
+	switch {
+	case path == "/dev/stdout":
+		writeCardTo(os.Stdout, data)
+	case path == "/dev/stderr":
+		writeCardTo(os.Stderr, data)
+	case path != "":
+		ioutil.WriteFile(path, data, 0644)
+	}
+}
+
+func writeCardTo(out io.Writer, data []byte) {
+	encoded := base64.StdEncoding.EncodeToString(data)
+	io.WriteString(out, "\u001B]1338;")
```

Comment:
please comment on this. It's hard to read as-is

Reply:
Not needed for this plugin. It comes from the boiler template. 

Do you think I should remove?

Reply:
Removed as discussed on daily meeting